homeassistant/components/media_player/sonos.py

```diff
@@ -351,11 +344,16 @@ def play_media(self, media_type, media_id, **kwargs):
             self._player.play_uri(media_id)
 
     @only_if_coordinator
-    def group_players(self):
+    def group_players(self, service):
```

Comment:
I changed it for consistency, snapshot/restore took a service parameter also, I'll refactor it.